test/unit/org/apache/cassandra/io/sstable/format/bti/RowIndexTest.java

```diff
@@ -89,12 +86,10 @@ public class RowIndexTest
     @Parameterized.Parameters()
     public static Collection<Object[]> generateData()
     {
-        return Arrays.asList(new Object[]{ Config.DiskAccessMode.standard, OSS50 },
-                             new Object[]{ Config.DiskAccessMode.mmap, OSS50 },
-                             new Object[]{ Config.DiskAccessMode.standard, OSS41 },
-                             new Object[]{ Config.DiskAccessMode.mmap, OSS41 },
-                             new Object[]{ Config.DiskAccessMode.standard, LEGACY },
-                             new Object[]{ Config.DiskAccessMode.mmap, LEGACY });
+        return Arrays.asList(new Object[]{ Config.DiskAccessMode.standard, latestVersion },
+                             new Object[]{ Config.DiskAccessMode.mmap, latestVersion },
+                             new Object[]{ Config.DiskAccessMode.standard, legacyVersion },
+                             new Object[]{ Config.DiskAccessMode.mmap, legacyVersion });
     }
```

Comment:
For Review:

In 5.0 `RowIndexWriter` uses `o.a.c.io.sstable.Version`, while `main` is using `ByteComparable.Version`.

I've kept the 5.0 `io.sstableVersion` here.

`latestVersion` and `legacyVersion` are defined as:

```
    private static final Version latestVersion = new BtiFormat(null).getLatestVersion();
    private static final Version legacyVersion = new BtiFormat(null).getVersion("aa");
```